sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -907,6 +907,12 @@ object SQLConf {
     .stringConf
     .createOptional
 
+  val FORCE_DELETE_TEMP_CHECKPOINT_LOCATION =
+    buildConf("spark.sql.streaming.forceDeleteTempCheckpointLocation")
+      .doc("When true, enable temporary checkpoint locations force delete.")
```

Comment:
nit: I'd like to have this to be a valid and more described sentence like other configurations, especially this feature is only documented here. One example for me, `When true, Spark always deletes temporary checkpoint locations for success queries.`

Reply:
Ur, @HeartSaVioR . That is the behavior before this PR.
The benefit of this configuration is to delete the locations when exception occurs.

Reply:
Ah yes my bad I meant `failed queries`. Anyway this option is only documented here so I'd just wanted to make sure there's clear description.